chpl/chpl-api/src/test/java/gov/healthit/chpl/web/controller/ProductControllerTest.java

```diff
@@ -111,4 +112,31 @@ public void testUpdateDuplicateChplProductNumberValidationException() throws Jso
         
         productController.updateProduct(request);
     }
+	
+	@Transactional
+	@Test
+	@Rollback(true)
+	public void testUpdateProductContactInformation() throws EntityRetrievalException, JsonProcessingException, EntityCreationException, InvalidArgumentsException, ValidationException {
```

Comment:
Does this test have anything to do with the feature in this PR? Or was it just added for some other reason?

Reply:
There was no test in this class to test when the product's contact information was being updated, so I added it.  This was the scenario called out in the ticket - updating a product's contact info caused an error.  I wrote the test before making the change, it failed.  Made the change and it passed.  I'm not sure what I did wrong here...

Reply:
I didn't say you did anything wrong. It's just that the code that changed doesn't seem to have any relationship to contact information changing, and that looked odd. If the test failed initially, then the code was changed, and the test passed, then that's TDD (or at least the first two steps). Unless you see anything that should need refactoring then it's good